easyrest/models/menu_item.py

```diff
@@ -26,7 +27,11 @@ class MenuItem(Base):
     description = Column(Text)
     ingredients = Column(Text)
     img = Column(Text)
+    price = Column(Float)
```

Comment:
Please never use `Float` for money, as it causes some precision lose during arithmetic operations, there is special type ([Numeric](https://docs.sqlalchemy.org/en/latest/core/type_basics.html#sqlalchemy.types.Numeric)) with fixed decimal point there. Do not forget to set correct scale.